lib/game.js

```diff
@@ -171,5 +173,15 @@ Game.prototype.instantiatePrefab = function(name) {
 	}.bind(this));
 	return id;
 };
-
-module.exports = Game;
+Game.prototype.registerPrefab = function (name,obj){
+	var prefabs = this.prefabs;
+	prefabs[name] = obj;
+};
+Game.prototype.importPrefabs = function(library){
+	var game = this;
```

Comment:
You don't need this variable. Just add a `.bind(this)` to the end of the anonymous function so `this.registerPrefab()` works inside of it.